lib/clients/GcpClient.py

```diff
@@ -0,0 +1,553 @@
+from .BaseClient import BaseClient
+from googleapiclient import discovery
+from google.oauth2 import service_account
+from google.cloud import storage
+from google.cloud.storage import Blob
+from .BaseClient import BaseClient
+from ..models.Snapshot import Snapshot
+from ..models.Volume import Volume
+from ..models.Attachment import Attachment
+import json
+import glob
+
+class GcpClient(BaseClient):
+    def __init__(self, operation_name, configuration, directory_persistent, directory_work_list, poll_delay_time,
+                 poll_maximum_time):
+        super(GcpClient, self).__init__(operation_name, configuration, directory_persistent, directory_work_list,
+                                        poll_delay_time, poll_maximum_time)
+
+        self.__gcpCredentials = json.loads(configuration['credentials'])
+        self.project_id = configuration['projectId']
```

Comment:
AFAIU, double underscore will mangle the attribute names of a class to avoid conflicts of attribute names between classes, see [understanding-the-underscore-of-python](https://hackernoon.com/understanding-the-underscore-of-python-309d1a029edc). So, it is not exactly meant for private access, but can be used so.

Reply:
ok